docs/reference/field-types/checkboxes.md

```diff
@@ -54,6 +55,13 @@ genres: {
 <!-- TODO: The following settings are likely to return, but are not yet implemented. -->
 <!-- |contextual | Boolean | false | If `true`, it will prevent the field from appearing in the editor modal | -->
 
+### `following`
+This option should be set to the name of a field or an array of field names that can be used to dynamically create choices based on the value of the followed field(s).
```

Comment:
It only makes sense to use this option when `choices` is a string, e.g. a method name on the server side to be invoked to generate the choices.

Reply:
Changed in all three documents.